python/lsst/ip/isr/isrTask.py

```diff
@@ -1225,3 +1225,50 @@ def getSaturation(self):
 
     def getSuspectLevel(self):
         return float("NaN")
+
+
+class IsrWrapperConfig(pexConfig.Config):
+    isr = pexConfig.ConfigurableField(target=IsrTask, doc="Instrument signature removal")
+
+## @addtogroup LSST_task_documentation
+## @{
+## @page IsrWrapperTask
+## @ref IsrWrapperTask_ "IsrWrapperTask"
+## @copybrief IsrWrapperTask
+## @}
+
+
+class IsrWrapperTask(pipeBase.CmdLineTask):
+    """Task to wrap the default IsrTask to allow it to be retargeted.
+
+    The standard IsrTask can be called directly from a command line
+    program, but doing so removes the ability of the task to be
+    retargeted.  As most cameras override some set of the IsrTask
+    methods, this would remove those data-specific methods in the
+    output post-ISR images.  This wrapping class fixes the issue,
+    allowing identical post-ISR images to be generated by both the
+    processCcd and isrTask code.
+    """
+    ConfigClass = IsrWrapperConfig
+    _DefaultName = "isrWrap"
+
+    def __init__(self, *args, **kwargs):
+        pipeBase.Task.__init__(self, *args, **kwargs)
```

Comment:
I think @natelust is or shortly will be going through the `Tasks` and making them use `super()` in places like this, so we can use multiple inheritance if we need to during the `CmdLineTask`->`PipelineTask` transition.  Might be good to go ahead and do that in new code right now.